src/main/scala/tile/BaseTile.scala

```diff
@@ -201,6 +201,13 @@ abstract class BaseTile private (val crossing: ClockCrossingType, q: Parameters)
   val resetVectorSinkNode = BundleBridgeSink[UInt](Some(() => UInt(visiblePhysAddrBits.W)))
   resetVectorSinkNode := resetVectorNode
 
+  /** Node for prefixing base addresses of MMIO slaves. */
```

Comment:
does this prefix work the same as the other (it is OR-ed in) or is it actually a prefix?

Reply:
Yeah similarly or'd. Open to a better name for describing this pattern but it seems to be what we are converging on at the wiring level.